OrbitGl/TimeGraph.cpp

```diff
@@ -211,6 +211,31 @@ void TimeGraph::PanTime(int a_InitialX, int a_CurrentX, int a_Width,
   NeedsUpdate();
 }
 
+void TimeGraph::HorizontallyScrollIntoView(const TextBox* text_box) {
+  const Timer& timer = text_box->GetTimer();
+
+  if (IsVisible(timer)) {
+    return;
+  }
+
+  // Where in the screen is going to be the center of the text box
+  double ratio = 0.3;
```

Comment:
Consider making this a parameter. Depending on the direction you are moving in, you might want the textbox be at 1/3 (when moving left) or at 2/3 (when moving right) (or maybe the other way around...).

Reply:
Ah, you already to that below if I understand right.